com.unity.netcode.gameobjects/Runtime/Messaging/MessagingSystem.cs

```diff
@@ -426,6 +432,13 @@ private bool CanSend(ulong clientId, Type messageType, NetworkDelivery delivery)
             return tmpSerializer.Length + headerSerializer.Length;
         }
 
+        internal unsafe int SendPreSerializedMessage<TMessageType>(in FastBufferWriter tmpSerializer, int maxSize, ref TMessageType message, NetworkDelivery delivery, ulong clientId)
```

Comment:
Adds a entrypoint taking a single `clientId` instead of a `PointerListWrapper`, because that class is private.